.circleci/config.yml

```diff
@@ -178,18 +178,22 @@ commands:
           no_output_timeout: 30m
       - check-3scale-templates-deployed-routes
   push-3scale-images-to-quay:
+    parameters:
+      components_imagestream_tag_name:
+        type: string
     steps:
       - docker-login
       - run:
           name: Push nightly images to quay.io
           command: |
             oc whoami --show-token | docker login -u $(oc whoami) --password-stdin 172.30.1.1:5000
             project=$(oc project -q)
+            imagestream_tag_name=<< parameters.components_imagestream_tag_name >>
```

Comment:
why local env var?

Reply:
because I prefer not to keep referencing the paramter the way `<< parameters.components_imagestream_tag_name >>`
on several lines

Reply:
why not? it is clear

Reply:
I don't want to repeat the reference in several places of the code (reuse the value) and I don't want to limit references to circlec-ci specific syntax as much as possible

Reply:
:woman_shrugging: